### DISCUSSION TITLE
Puzzle 34 Error in Solution 3

### DISCUSSION THREAD

**Discussion by 2b4fITin** [smpetersgithub/AdvancedSQLPuzzles#39] [General]
Seems this is wrong: 'WHERE CustomerID <> 1001 OR Amount <> 50;'
IMHO this is correct: 'WHERE CustomerID = 1001 AND Amount = 50;'

**Comment by smpetersgithub** [comment]
Thank you!  I will check on this.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: 2b4fITin ***@***.***>
Sent: Thursday, May 30, 2024 12:14:48 AM
To: smpetersgithub/AdvancedSQLPuzzles ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [smpetersgithub/AdvancedSQLPuzzles] Puzzle #34 Error in Solution 3 (Discussion #39)


Seems this is wrong: 'WHERE CustomerID <> 1001 OR Amount <> 50;'
IMHO this is correct: 'WHERE CustomerID = 1001 AND Amount = 50;'

**Comment by smpetersgithub** [comment]
This has been fixed.  Thank you!  Please let me know of any errors that you find.
